tests/conftest.py

```diff
@@ -1,10 +1,12 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 
+# pylint: disable=missing-module-docstring
```

Comment:
Alternatively, it could just be `"""Fixtures for tests"""`

Reply:
Added, thanks!